From ceae3b58f1460806b6cb7cc94b5bcbfa0b05fc35 Mon Sep 17 00:00:00 2001
From: Peter Martienssen <peter.martienssen@liquify-consulting.de>
Date: Thu, 13 Jan 2022 09:41:10 +0100
Subject: [PATCH 1/4] Added VC MIPI driver files to .gitignore

---
 .gitignore | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.gitignore b/.gitignore
index 70580bdd352c..02b7d3700c1f 100644
--- a/.gitignore
+++ b/.gitignore
@@ -146,3 +146,6 @@ x509.genkey
 
 # Clang's compilation database file
 /compile_commands.json
+
+# VC MIPI driver files
+*vc_mipi*
\ No newline at end of file
-- 
2.25.1

